PortMaster/retrodeck/PortMaster.txt

```diff
@@ -23,7 +24,7 @@ echo "Starting PortMaster."
 chmod -R +x .
 
 #check RetrodDECK installed and read location of roms folder
-if [[ -e "${HOME}/.var/app/net.retrodeck.retrodeck/config/retrodeck/retrodeck.cfg" ]]; then
+if [[ -e "/app/bin/retrodeck.sh" ]]; then
```

Comment:
Actually when /app/libexec/global.sh is sourced, roms folder is just $roms_folder, PortMaster games are in $roms_folder/portmaster for us, and not inside ports as it's a different system.
PortMaster will be its own "system" in ES-DE.